Review of the Getting Started content (#404)
neo4j/docs-getting-started: modules/ROOT/pages/tutorials/cypher-sql.adoc

:page-ad-underline-role: button
:page-ad-underline: Learn more

This tutorial shows anyone familiar with SQL how to understand Cypher and write the equivalent Cypher queries using the Northwind dataset.

[NOTE]

> To know more what are the differences between the two query languages, refer to the link:{docs-home}/cypher-manual/current/introduction/cypher-overview/#_cypher_and_sql_key_differences[Cypher documentation].

Reviewer: ```suggestion
====
To learn more about the differences between the two query languages, refer to the link:{docs-home}/cypher-manual/current/introduction/cypher-overview/#_cypher_and_sql_key_differences[Cypher documentation].
====
```

[#cypher-sql-northwind]
== Load the Northwind dataset

The Northwind dataset represents the data storage of a retail application.
It contains customers, products, orders, employees, shippers, and categories as well as their interactions.
Follow these steps to start working with this dataset:

> . link:{docs-home}/deployment-options[Create a Neo4j instance].

Reviewer: This step should not link anywhere but contain instructions for how to create an instance. Either we decide that this is a tutorial for Aura users or for on-prem users. We could also tab it to provide both.

. Import the Northwind dataset.

> .. In case you are using Aura, click on the hat icon on the top-right side menu and follow the _Query fundamentals_ guide up to step 2.

Reviewer: ```suggestion
.. If you are using Aura, click on the hat icon on the top-right side menu and follow the _Query fundamentals_ guide up to step 2.
```

+
image::northwind_dataset_load.png[]
.. If you are using Neo4j Browser, run `:guide northwind` in the query area.
+
image::browser-northwind.png[]

> .. Another option, functional both in Aura and Browser, is to xref:tutorials/import-relational-and-etl.adoc[import the dataset as a _.zip_ file].

Reviewer: Let's keep it simple and offer one way of importing.

> Your graph should now look like this:

Reviewer: This is the data model, not the graph. Also, this presumes that the user is using Workspace. If they're using Browser they would have to run CALL db.schema.visualization to get the same.

image::northwind.png[]

== Start querying

> A first, basic query that you can do is t

Reviewer: ```suggestion
A first basic query to do is to find and return records.
```

> _Select and return records_

Reviewer: ```suggestion

```

> To select and return records in SQL, select everything from the `products` table.

Reviewer: ```suggestion
To find and return records in SQL, you need to `SELECT` what you want to find.
For example, select everything from the `products` table:
```

[source, plsql]
----
SELECT p.*
FROM products as p;
----

> Relational databases store data in tables with a fixed structure (schema), each column having a name, type, length, constraints, etc.

Reviewer: ```suggestion
Relational databases store data in tables with a fixed structure (schema), where each column has a name, type, length, constraints, etc.
```

> On the other hand, graph databases store data in a graph with nodes and relationships between them.

Reviewer: ```suggestion
Graph databases store data in a graph with nodes and relationships between them.
```
